packages/twitch/src/API/Kraken/Team/TeamWithUsers.ts

```diff
@@ -0,0 +1,23 @@
+import Team, {TeamData} from './Team';
```

Comment:
As CI already noted, spacing is off. Please make sure that the lint and build jobs don't fail before pushing.